src/runtime/test.rs

```diff
@@ -29,3 +29,10 @@ fn test_linux_device_cgroup_to_string() {
         .expect("build device cgroup");
     assert_eq!(ldc.to_string(), "a 1:9 rwm");
 }
+
+#[test]
+fn test_load_sample_spec() {
+    let fixture_path = std::path::PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+        .join("src/runtime/test/fixture/sample.json");
+    Spec::load(fixture_path).unwrap();
```

Comment:
How about  this to make it clearer that we do testing?
```suggestion
    assert!(Spec::load(fixture_path).is_ok());
```

Reply:
Exactly what I did before lol. The problem is that in the case it fails, `.is_ok()` will swallow the error, since `assert!` only cares about boolean value. Ideally we would have something like `assert_error`, but since we don't, `.unwrap()` is sort of the next best thing that will fail and shows me what the error is actually failing.

Reply:
I think it is better to use `assert`.  I agree with your suggestion.